.github/workflows/actions.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       HAS_TOKEN: ${{ secrets.DOCKERHUB_USERNAME != '' &&  secrets.DOCKERHUB_TOKEN != ''}}
 
     container:
-      image: cspiffe/grpc-build:1.34.0
+      image: cspiffe/federation-tests:v1
```

Comment:
Shouldn't the `grpc-build` image be updated rather than replaced with ` federation-tests`?